Add reciepe for kapacitor-el

Manoj321 opened the pull request:
### Brief summary of what the package does

This package provides a magit interface to kapacitor. It uses magit's popup system for navigation.

### Direct link to the package repository

https://github.com/Manoj321/kapacitor-el

### Your association with the package

I'm the author and maintainer.

### Relevant communications with the upstream package maintainer

**None needed**

### Checklist

Please confirm with `x`:

- [X] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses). 
- [X] I've read [CONTRIBUTING.md](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md)
- [X] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [X] My elisp byte-compiles cleanly
- [X] `M-x checkdoc` is happy with my docstrings
- [X] I've built and installed the package using the instructions in [CONTRIBUTING.md](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md)

alphapapa commented:
Hi,

A little feedback:

1.  Generally those 3 source files could easily be in a single file.  There's no need to split them up like that, because they're so small.
2.  The package headers are incorrect.  Please see e.g. https://github.com/alphapapa/emacs-package-dev-handbook#template
3.  There are miscellaneous minor issues, like typos and spelling errors in docstrings, incorrect spacing (e.g. `(defun name() ...`), etc.

Manoj321 commented:
Thanks for the feedback!

1. I plan to add more functionality later thats why I have 3 separate files.
2. corrected package headers
3. Addressed in consistency in spacing

alphapapa commented:
> I plan to add more functionality later thats why I have 3 separate files.

What I mean is, it's not customary to split up Emacs packages like that, e.g. by putting `defvar`s in a separate file, especially when each file is so small.  It's not necessarily wrong, but it's not what's expected, so it's a bit more trouble for other developers to work with your code.  Elisp isn't C.  :)

> corrected package headers

There are still missing headers, e.g. no dependencies are declared.

> Addressed in consistency in spacing

What I meant was, there *should be* spaces between function names and argument lists.  Again, Elisp isn't C.  :)  Here are some helpful resources:

* https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html
* https://github.com/bbatsov/emacs-lisp-style-guide

Manoj321 commented:
Thanks for the pointers, I have made all the suggested modification please have a look.

alphapapa commented:
Looks good, thanks.  You might also like to put a screenshot in the readme to help users understand what it does.

Manoj321 commented:
Yep, working on it. Thanks.

sent from Android.

On Sun, Aug 5, 2018, 6:02 PM alphapapa <notifications@github.com> wrote:

> Looks good, thanks. You might also like to put a screenshot in the readme
> to help users understand what it does.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/melpa/melpa/pull/5652#issuecomment-410562318>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AB_GyhY3Dev7LNrJAa2bRl2NmyrrpgCjks5uN5WMgaJpZM4VlOcI>
> .
>

Manoj321 commented:
Can someone please merge this PR?
